sublime-package.json

```diff
@@ -517,25 +547,51 @@
                             "warning",
                             "error"
                           ]
+                        },
+                        "reportUnusedCallResult": {
+                          "type": "string",
+                          "description": "Diagnostics for call expressions whose results are not consumed and are not None.",
+                          "default": "none",
+                          "enum": [
+                            "none",
+                            "information",
+                            "warning",
+                            "error"
+                          ]
+                        },
+                        "reportUnusedCoroutine": {
+                          "type": "string",
+                          "description": "Diagnostics for call expressions that return a Coroutine and whose results are not consumed.",
+                          "default": "error",
+                          "enum": [
+                            "none",
+                            "information",
+                            "warning",
+                            "error"
+                          ]
+                        },
+                        "reportUnsupportedDunderAll": {
+                          "type": "string",
+                          "description": "Diagnostics for unsupported operations performed on __all__.",
+                          "default": "warning",
+                          "enum": [
+                            "none",
+                            "information",
+                            "warning",
+                            "error"
+                          ]
                         }
                       }
                     },
-                    "python.analysis.extraPaths": {
-                      "type": "array",
-                      "description": "Additional import search resolution paths.",
-                      "items": {
-                        "type": "string"
-                      }
-                    },
                     "python.analysis.logLevel": {
                       "type": "string",
                       "default": "info",
                       "description": "Specifies the level of logging for the Output panel",
                       "enum": [
-                        "error",
-                        "warning",
-                        "info",
-                        "trace"
+                        "Error",
+                        "Warning",
+                        "Information",
+                        "Trace"
```

Comment:
You could leave it the way it was because it was more consistent with other options that take lowercase values.

The actual case doesn't matter for pyright as it [lowercases the value](https://github.com/microsoft/pyright/blob/c03e8c309e936d970d2df7f4ba858ff438772bc5/packages/pyright-internal/src/languageServerBase.ts#L1101-L1122).